systems/plants/RigidBodyManipulator.h

```diff
@@ -284,6 +284,8 @@ class DLLEXPORT_RBM RigidBodyManipulator
   std::string getBodyOrFrameName(int body_or_frame_id);
   //@param body_or_frame_id   the index of the body or the id of the frame.
 
+  int parseBodyOrFrameID(const int body_or_frame_id, Matrix4d* Tframe = nullptr);
```

Comment:
not sure how I feel about making this public.  it makes some assumptions internally... @tkoolen -- what do you think?